tests/support/util.tcl

```diff
@@ -1225,3 +1225,21 @@ proc bp {{s {}}} {
         puts $res
     }
 }
+
+proc valkey_version_at_least {min_version} {
+    set i [r info]
+    regexp {valkey_version:(.*?)\r\n} $i - valkey_version
+    set actual [split $valkey_version .]
+    set required [split $min_version .]
+
+    for {set i 0} {$i < 3} {incr i} {
+        set a [expr {[llength $actual] > $i ? [lindex $actual $i] : 0}]
+        set b [expr {[llength $required] > $i ? [lindex $required $i] : 0}]
+        if {$a > $b} {
+            return 1
+        } elseif {$a < $b} {
+            return 0
+        }
+    }
+    return 1 ;# exact match
+}
```

Comment:
Why so complex? You didn't like my example code I posted before?

Instead of regexp + string split + loop, we can use only regexp and a condition.

I can adjust my suggestion to call the `INFO` command:

```suggestion
proc valkey_version_at_least {min_version} {
    set i [r info server]
    regexp {valkey_version:(\d+)\.(\d+)\.(\d+)} $i - major minor patch
    regexp {^(\d+)\.(\d+)\.(\d+)} $min_version - min_major min_minor min_patch
    return [expr {$major > $min_major ||
                  ($major == $min_major && $minor > $min_minor) ||
                  ($majro ==  $min_major && $minor == $min_minor && $patch >= $min_patch)}]
}
```

I can adjust it to